components/shared/workCard/helper.ts

```diff
@@ -1,11 +1,11 @@
 import { flatten } from "lodash"
 
-import { Cover } from "@/lib/cover-service-api/model"
 import { SearchWithPaginationQuery, WorkTeaserFragment } from "@/lib/graphql/generated/fbi/graphql"
 import { filterFalsyValuesFromArray } from "@/lib/helpers/arrays"
 
-import { CoverImageUrls } from "./../../../lib/cover-service-api/model/coverImageUrls"
 import { GeneralMaterialTypeCode } from "./../../../lib/graphql/generated/fbi/graphql"
+import { Cover } from "./../../../lib/rest/cover-service-api/generated/model"
+import { CoverImageUrls } from "./../../../lib/rest/cover-service-api/generated/model/coverImageUrls"
```

Comment:
I generally like imports routes to be set from project root. 

Something like this
```suggestion
import { CoverImageUrls } from "@/*??*/*??*/lib/rest/cover-service-api/generated/model/coverImageUrls"
```

It becomes easier to move files around and update imports if the import route is not rooted in the actual file. Consider doing this.

Reply:
Wonderful suggestion!!! THX!!!

Reply:
Great i tend to sometimes forget myself. Please also correct me 😊